molecule/default/molecule.yml

```diff
@@ -10,12 +10,28 @@ platforms:
     privileged: true
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  - name: minio-fedora-27
```

Comment:
Why do we depend on externally build docker images?
Can't something like that be implemented? 

https://github.com/atosatto/ansible-prometheus/blob/7b286ce625ae96467e5363378a60cc55c6103069/molecule/prometheus-latest/molecule.yml#L15-L25

I know that it's not super efficient, but it makes the CI self-contained within the role.

Reply:
I think the test time efficiency was the goal. @paulfantom?

Reply:
Yes, I use those images to reduce time needed for CI pipeline. Those are automatically built from this repo https://github.com/paulfantom/dockerfiles

Reply:
@atosatto self-contained images are ok, but consider testing this across 3 versions of ansible and 5-6 distros, this will cause to build those images 15-18 times for every new commit. This takes a lot of time and doesn't scale well. Also this way when something breaks in one repo, we can quickly fix it for all of them.

Reply:
I did the same in the past (https://github.com/atosatto/dockerfiles) and then switched to self-contained images built as part of the CI project.

I am fine with the change for now. We can always switch back to Docker images built as part of the CI in a next stage.